.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Report a bug
+about: Report a bug to help us improve.
+title: "[what is the issue?] in [where is the issue?]"
+labels: "status:triage"
+assignees: ""
+---
+
+**Describe the bug; what happened?**
+
+**What are the steps to reproduce the issue?**
+
+**What behavior did you expect?**
+
+**If applicable, provide screenshots:**
+
+**In what environment did you see the issue?**
+
+- OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
+- Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
```

Comment:
I think we can remove Browser

Reply:
I think Pavel, we can leave it like this. Developers can indicate browser in case they are talking about issues like video stream is displayed on chrome when joined call from Android. Please let me know if we want to remove it.